CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,73 @@
+# Code of Conduct
+
+In order to foster a welcoming, positive, and rewarding experience, we as
+community members take an oath to uphold this code of conduct.
+
+*Isn't decent behavior is already expected?* We want to bring the best we can
+to each other as peers, and explicitly documenting our expectations lets us set
+the bar high for ourselves.
+
+## Be excellent to each other
+
+"Excellent" means we:
+
+- **are radically inclusive** to existing members and newcomers looking to
+  learn or participate. Some may raise questions that have been addressed
+  before. Kindly point them to existing discussions. Everyone is new to a
+  project once.
+- **respect** your abilities, interests, and personal differences.  Our
+  different talents documenting, testing, coding, reviewing, reporting, and/or
+  evagenlizing are all valuable to the project.
+- **make concise and actionable points** to respect your time as a volunteer.
+  We respect each other's time by keeping the signal-to-noise ratio high,
+  staying on-topic, and being a good house-guest: every visit we strive to
+  leave the place in better shape than when we arrived.
+- **are kind** and **assume positive intent**, even when we disagree while
+  working towards consensus and achieving the project's technical goals.
+- **educate and illuminate** others with something we know more about. We were
+  all beginners once, and can teach and learn through feedback and sharing.
+- **are open to feedback** from others, be it technical points,  interpersonal
+  tips, or language/wording suggestions.
```

Comment:
There's an extra space before interpersonal

Reply:
Good eye :-) fixed in 8384250a